automation/test.sh

```diff
@@ -284,6 +287,8 @@ EOF
   export KUBEVIRT_E2E_FOCUS=Windows
 elif [[ $TARGET =~ (cnao|multus) ]]; then
   export KUBEVIRT_E2E_FOCUS="Multus|Networking|VMIlifecycle|Expose|Macvtap"
+elif [[ $TARGET =~ sig-network ]]; then
+  export KUBEVIRT_E2E_FOCUS="\[sig-network]"
```

Comment:
You use the escape char (`\`) once, why? I would expect it is either needed or it is not.

Reply:
it doesn't match without it, its a regex, that need it, the other `]` doesn't need to be escaped, 
i tested all the combinations

Reply:
I know why it is needed, I do not understand why you did not add it to the closing one for completeness.
It looks odd and raises questions.